### DISCUSSION TITLE
`$sub_frame` filter not found

### DISCUSSION THREAD

**Discussion by uBlock-user** [uBlockOrigin/uBlock-issues#2302] [General]
STR --

- Add `@@||twitch.tv^$csp` and `*$csp=worker-src 'none',1p` to my filters. 

- Open the logger in a new window and browse to `https://www.twitch.tv/` in a new tab.

- Switch to the logger, click on `@@||twitch.tv^$sub_frame,1p` and notice that filter is not recognised in any filter lists, because there's no support for `$sub_frame` directive(shown as invalid in my filters)

@gorhill Will you add support for `$sub_frame` or add it as an alias to `$frame` ?

**Comment by gorhill** [comment]
Related to changes I made for DNR, I need to fix this by serializing to `frame` as before.

**Comment by uBlock-user** [comment]
> `$sub_frame` directive(shown as invalid in my filters)

Will it also address that ^^^ ?

**Reply by gorhill** [reply]
Yes.

**Reply by uBlock-user** [reply]
My filters still shows `$sub_frame` as invalid in 1.44.5b8.

**Reply by gorhill** [reply]
I don't see any instance of `sub_frame` in the logger.

**Reply by uBlock-user** [reply]
See below.

![Capture](https://user-images.githubusercontent.com/21290713/192835649-19e40628-dfe2-452f-b7fc-d11c8fdffc55.PNG)

**Reply by gorhill** [reply]
This has never worked, why do you want this to work now?

**Reply by uBlock-user** [reply]
Good to know, I thought it worked before and got regressed, hence the suggestion to alias it to `$frame`, nvm.
